developer_guide/installation/system_requirements/system_install_osx_1011.rst

```diff
@@ -0,0 +1,276 @@
+
+.. note::
+
+    The `--with-httpd24` option installs the package httpd24 (apache) automatically and compiles the libphp5.so extension for apache. If you want to
+    use the OS X built-in apache version, you need to unlink the package using the following command: `brew unlink httpd24` (This command does not delete
+    the libphp5.so extension that you will need to include your apache configuration).
+
+* Check that PHP 5.6 is now your current PHP version with:
+
+.. code-block:: bash
+
+    $ php -v
+
+* Now you can directly continue by :ref:`choosing_product_storage`.
+
+Choosing the product storage
+****************************
+
+.. include:: /reference/technical_information/choose_database.rst.inc
+
+Based on this formula, either you need a :ref:`mongodb-install-1604`, either you can directly go to the :ref:`system-configuration-1604` section.
+
+.. _mongodb-install-1604:
+
+MongoDB Installation (optional)
+*******************************
+
+PHP 5.6 (supported)
+^^^^^^^^^^^^^^^^^^^
+
+* Install MongoDB server and PHP driver
+
+.. note::
+
+    Akeneo PIM will not work with MongoDB 3.*. *The supported versions are 2.4 and 2.6*.
+
+.. code-block:: bash
+
+    $ brew update
+    $ brew install mongodb24
+    $ brew install php56-mongo
+
+Check that mongodb has been correctly installed by listing homebrew services:
+
+.. code-block:: bash
+
+    $ brew services list
+
+You can run the mongodb server with:
+
+.. code-block:: bash
+    $ brew services mongodb24 start
+
+System configuration
+--------------------
+
+You now have a system with the right versions of Apache, PHP and MySQL. The next step is to configure them to be able to run an Akeneo PIM instance.
+
+MySQL
+*****
+
+* Create a dedicated MySQL database and a dedicated user (called akeneo_pim) for the application
+
+.. code-block:: bash
+
+    $ mysql -u root -p
+    mysql> CREATE DATABASE akeneo_pim;
+    mysql> GRANT ALL PRIVILEGES ON akeneo_pim.* TO akeneo_pim@localhost IDENTIFIED BY 'akeneo_pim';
+    mysql> EXIT
+
+PHP
+***
+
+
+* Setup *php.ini* file ``/usr/local/etc/php/5.6/php.ini``
+
+.. note::
+
+    If you have several php versions on your machine installed with homebrew, these files can be located in ``/usr/local/etc/php/x.x/php.ini``.
+
+.. code-block:: yaml
+
+    $ sudo vim /etc/php5/cli/php.ini
+    memory_limit = 768M
+    date.timezone = Etc/UTC
+
+.. note::
+
+    Use the time zone matching your location, for example *America/Los_Angeles* or *Europe/Berlin*. See http://www.php.net/timezones for the list of all available timezones.
+
+Apache
+******
+
+Setting up the permissions
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To avoid spending too much time on permission problems between the CLI user and the Apache user, a good practice is to use the same user for both of them.
+
+.. warning::
+
+    This configuration is aimed to easily set up a development machine.
+    **It is absolutely not suited for a production environment.**
```

Comment:
Should we really ask ourselves if an OSX server configuration is suited for production?
Can you tell how many OSX webservers does run on OSX ?

:trollface: